oracle/src/oracleimporter.h

```diff
@@ -7,11 +7,35 @@
 #include <utility>
 
 // many users prefer not to see these sets with non english arts
-// as a solution we remove the date property on these sets
-// that way they will be sorted last by default
-// this will cause their art to not get priority over english cards
-// users will still be able to find these sets and prioritize them manually
 const QStringList nonEnglishSets = {"4BB", "FBB", "PS11", "PSAL", "REN", "RIN"};
+const QMap<QString, CardSet::Priority> setTypePriorities{
```

Comment:
Does this account for all types at this point?

Reply:
To my knowledge, yes, unless additional types have been added since I wrote this.